examples/file_dialog.py

```diff
@@ -1,11 +1,10 @@
 """FileDialog with magicgui."""
```

Comment:
This example isn't running for me, screen short of flashes and then code is done

Reply:
fixed